recipes/cmrt/build.sh

```diff
@@ -0,0 +1,6 @@
+# https://01.org/linuxgraphics/documentation/build-guide-0
+
+export CFLAGS="${CFLAGS} -lrt"
+export CXXFLAGS="${CXXFLAGS} -lrt"
+./autogen.sh --prefix=$PREFIX
+make -j$(nproc) install
```

Comment:
No make check?

Reply:
should I use `make check` for most packages?